src/Components/Cards/VerticalCard.jsx

```diff
@@ -108,52 +130,62 @@ export const VerticalCard = ({ videoDetail }) => {
   }, [watchlater, videoDetail._id, setWatchlaterButton, setLikeButton]);
 
   return (
-    <div className="card-vertical video-card-vertical">
+    <div className={cardOrientation}>
       <div className="card-img-container">
         <Link to={`/videos/${videoDetail._id}`}>
           <img src={thumbnails.medium.url} alt="thumbnail" loading="lazy" />
         </Link>
+        {onHistoryPage && (
+          <IconButton
+            onClick={deleteFromHistoryHandler}
+            btnClassName="btn icon-btn-sm history-delete-btn"
+            icon="fas fa-times "
+          />
+        )}
```

Comment:
could have used ternary operator instead of short circuiting

Reply:
I am following the practice to use ternary if it's not returning true or false values.